### PR TITLE
Alkwa/move util func to common

### PR DESCRIPTION
**Pull request by alkwa-msft**
# What
Moving a common util function from the composites directory to the acs-ui-common packlet.

# Why
There was an optimization suggested in the discussion below to reduce the bundle package size for those only building for a  chat experience
https://github.com/Azure/communication-ui-library/discussions/3208

### CODE REVIEW

**Review by dmceachernmsft — APPROVED** [review]

File: packages/acs-ui-common/review/beta/acs-ui-common.api.md [hunk 1]
```diff
@@ -51,6 +51,9 @@ export interface _IObjectMap<T> {
     [key: string]: T;
 }
 
+// @public
```
Comment (JamesBurnside) [inline packages/acs-ui-common/review/beta/acs-ui-common.api.md:54]: this should be `@internal`

Comment (alkwa-msft) [reply packages/acs-ui-common/review/beta/acs-ui-common.api.md:54]: will make this change

**Review by JamesBurnside — APPROVED** [review]

**Review by PorterNan — APPROVED** [review]
https://github.com/Azure/communication-ui-library/pull/3225
There is a PR opened for this to get a better production-like treeshaked bundle - so we can see the real number change from the comment once it is in

**Comment by JamesBurnside** [comment]
Wow! Yeah lets check this in!
image

**Pull request merged**